src/components/Input.js

```diff
@@ -28,6 +36,9 @@ const Input = () => {
       addBook();
     }
   };
+  // const store = useSelector((state) => state);
+
+  // dispatch(thunkFunction());
 
```

Comment:
- [ ] **Optional:** Kindly remove commented code from your codebase. It is considered bad practice to commit commented code. And it also makes code hard to read.